conf/app.routes

```diff
@@ -56,6 +56,11 @@ GET           /handlebarstemplates/:handlebarsTemplateId
 GET           /handlebarstemplates/:handlebarsTemplateId/raw                                            uk.gov.hmrc.gform.handlebarstemplate.HandlebarsTemplateController.getRaw(handlebarsTemplateId: HandlebarsTemplateId)
 DELETE        /handlebarstemplates/:handlebarsTemplateId                                                uk.gov.hmrc.gform.handlebarstemplate.HandlebarsTemplateController.delete(handlebarsTemplateId: HandlebarsTemplateId)
 
+GET           /handlebars-schemas                                                                        uk.gov.hmrc.gform.handlebarstemplate.HandlebarsSchemaController.allIds
+POST          /handlebars-schemas/:handlebarsSchemaId                                                    uk.gov.hmrc.gform.handlebarstemplate.HandlebarsSchemaController.upsert(handlebarsSchemaId: HandlebarsSchemaId)
+GET           /handlebars-schemas/:handlebarsSchemaId                                                    uk.gov.hmrc.gform.handlebarstemplate.HandlebarsSchemaController.get(handlebarsSchemaId: HandlebarsSchemaId)
+DELETE        /handlebars-schemas/:handlebarsSchemaId                                                    uk.gov.hmrc.gform.handlebarstemplate.HandlebarsSchemaController.delete(handlebarsSchemaId: HandlebarsSchemaId)
+
```

Comment:
There is one extra spavce before `uk.gov.hmrc.` compared to the lines above or below.